tests/test_events/test_events_ratios.py

```diff
@@ -60,6 +64,36 @@ def test_event_checkin_and_cancellation_ratio_per_room(self, mock_get_json):
             event_ratio_per_room_response
         )
 
+    @patch("helpers.calendar.analytics_helper.get_events_within_datetime_range",
+           spec=True)
+    def test_event_checkin_and_cancellation_single_room(self, mock_get_json):
+        """
+        Test that an admin is able to get the ratio of checkins to bookings
+        for a single room
+        """
+        mock_get_json.return_value = get_events_mock_data()
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            event_ratio_single_room_query,
+            event_ratio_single_room_response
+        )
+
+    @patch("helpers.calendar.analytics_helper.get_events_within_datetime_range",
+           spec=True)
+    def test_event_checkin_cancellation_single_room_wrong_id(self,
+                                                             mock_get_json):
+        """
+        Test that an error should be thrown when the room id provided doesn not
```

Comment:
Kindly change this docstring to have a more clear description.  An example is `Test an admin cannot get the ratio of check-ins to bookings for a single room with invalid room id`.